mmpose/utils/hooks.py

```diff
@@ -0,0 +1,66 @@
+import functools
+
+
+class Hook:
```

Comment:
there was a Hook class in mmcv https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/hook.py, 
to avoid confusion, how about define OutputsHook directly?

Reply:
and `OutputsHook` -> `OutputHook`